scripts/check_go_version.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+fix=0
+args="$(getopt -o "" -l fix -- "$@")"
+eval set -- "$args"
+while true; do
+	case "$1" in
+	--fix)
+		fix=1
+        shift
+		;;
+	--)
+		shift
+		break
+		;;
+	*)
+		error "Unrecognized option: $1"
+		;;
+	esac
+done
```

Comment:
Since this won't work on macOS, may I suggest a simple:

```bash
if [[ " $* " == *" --fix "* ]]; then
```

? 😄

Reply:
Is there a significance to the extra spaces?

Reply:
Yup, it's a trick so it can match args exactly, _and_ in any placement (start, middle, end).